assets/styles/importer.less

```diff
@@ -217,4 +216,24 @@ td h3 {
 
 .optional {
   font-weight: normal;
-}
\ No newline at end of file
+}
+
+.ban-message {
```

Comment:
Why did you move these?

Reply:
Figured it'd be nicer to have all the "misc"  CSS all in one place.